src/mist/io/methods.py

```diff
@@ -3891,20 +3920,21 @@ def probe_ssh_only(user, cloud_id, machine_id, host, key_id='', ssh_user='',
     }
 
 
-def ping(host):
-    ping = subprocess.Popen(
-        ["ping", "-c", "10", "-i", "0.4", "-W", "1", "-q", host],
-        stdout=subprocess.PIPE
-    )
-    ping_out = ping.stdout.read()
-    ping.wait()
-    return parse_ping(ping_out)
+def ping(host, user=None):
```

Comment:
this should be done inside super_ping

actually the code of super_ping should be merged here